neonvm-kernel/linux-config-amd64-6.6.64

```diff
@@ -286,6 +286,19 @@ CONFIG_PERF_EVENTS=y
 CONFIG_SYSTEM_DATA_VERIFICATION=y
 # CONFIG_PROFILING is not set
 CONFIG_TRACEPOINTS=y
+
+#
+# Kexec and crash features
+#
+CONFIG_CRASH_CORE=y
+CONFIG_KEXEC_CORE=y
+CONFIG_KEXEC=y
```

Comment:
Weird that this is enabled. I guess it already was, but maybe worth disabling? (either way, doesn't need to be this PR)

Reply:
https://cateee.net/lkddb/web-lkddb/KEXEC.html

Looks like it was marked as (EXPERIMENTAL) before 6.6, and probably it's enabled by default starting from 6.6 because it's no longer experimental?